### DISCUSSION TITLE
img2img always freezes after one or two generations

### DISCUSSION THREAD

**Discussion by FlayerTwo** [lshqqytiger/stable-diffusion-webui-directml#266] [Q&A]
I'm using an RX 6800 and I've been able to get the txt2img working somewhat consistently using the commonly mentioned arguments. But img2img always only manages to generate one or two results before becoming completely unresponsive. No errors in command prompt, no errors in the web UI. Just nothing, until I restart it and am once again able to generate one or two img2img results before the next freeze. Is this a known issue? It was mentioned in #264 and I'm wondering whether something in 1.6.0 could be the culprit.

**Comment by Cimm-Yeoman** [comment]
What resolution are you using for img2img? My GPU is a lot older than yours but I found I ran into this issue when I pushed the resolution too high. Also, sometimes the common arguments "--no-half" and/or "--precision full" can actually result in GPU performance issues, so try removing those and see if it helps with the freezing.
